django/db/migrations/migration.py

```diff
@@ -184,15 +183,24 @@ def suggest_name(self):
         name to avoid VCS conflicts if possible.
         """
         name = None
-        if len(self.operations) == 1:
-            name = self.operations[0].migration_name_fragment
-        elif (
-            len(self.operations) > 1 and
-            all(isinstance(o, operations.CreateModel) for o in self.operations)
-        ):
-            name = '_'.join(sorted(o.migration_name_fragment for o in self.operations))
-        if name is None:
-            name = 'initial' if self.initial else 'auto_%s' % get_migration_name_timestamp()
+        if self.initial:
+            return "initial"
+
+        raw_fragments = (op.migration_name_fragment for op in self.operations)
+        fragments = [name for name in raw_fragments if name]
+
+        if not fragments:
+            return f"auto_{get_migration_name_timestamp()}"
+        name = fragments[0]
+        left_out = len(fragments) - 1
+        for fragment in fragments[1:]:
+            new_name = name + f"_{fragment}"
+            if len(new_name) > 52:
+                break
+            name = new_name
+            left_out -= 1
+        if(left_out > 0):
+            name = name + f"_+{left_out}"
```

Comment:
We shouldn't use `+` in filenames, moreover the number of additional operations doesn't have much value.

`_and_more` sounds like a good marker (see [comment](https://github.com/django/django/pull/12799#issuecomment-636694424)).

Reply:
> What about `_16_more`(16 is the number of operations)?

I don't see much value in this format,  you still need to open the file to find out more.